bom/pom.xml

```diff
@@ -195,6 +186,23 @@
                 <scope>import</scope>
                 <type>pom</type>
             </dependency>
+            <!-- We need to fix this specific Jackson module version due to dependency management issue -->
```

Comment:
I think this comment should be updated with something like:

> The Jackson BOM imports version 2.9.0 and we want all the versions to be in sync. Especially, 2.9.1 adds an automatic module name.

Reply:
done